assets/wpuf/js/frontend-form.js

```diff
@@ -462,10 +462,10 @@
             WP_User_Frontend.removeErrors(self);
             WP_User_Frontend.removeErrorNotice(self);
 
-            // ===== Validate: Text and Textarea ========
-            var required = self.find('[data-required="yes"]:visible');
+            // Validate: Required Fields
+            var validate = self.find('[data-type]');

```

Comment:
Why the change?

What was wrong with `self.find('[data-required="yes"]:visible');`?

Reply:
The data-required was causing the validation scripts to only run when a field was required. But wouldn't we want to validate all fields whether or not they are required? We would want to validate any field that could have an entry.

Reply:
I would say, for now, if it's not broken, don't fix it.

The change you made here, it could cause some unexpected issues with other parts of the code... Or, if someone upgrades, a form on their site could stop working... How many field types are there? Are you sure is field type is correctly validated?

Reply:
This has been changed back for now